.evergreen/run-tests.sh

```diff
@@ -258,7 +258,8 @@ python -c 'import sys; print(sys.version)'
 # Only cover CPython. PyPy reports suspiciously low coverage.
 PYTHON_IMPL=$($PYTHON -c "import platform; print(platform.python_implementation())")
 if [ -n "$COVERAGE" ] && [ "$PYTHON_IMPL" = "CPython" ]; then
-    python -m pip install pytest-cov "coverage"
+    # coverage 7.3 dropped support for Python 3.7, keep in sync with combine-coverage.sh.
+    python -m pip install pytest-cov "coverage<7.3"
```

Comment:
Could we see a world where we have a version of python or dependent libraries that's incompatible with coverage<7.3? I don't imagine that's likely considering the python release schedules, and our support timelines but just curious.

Reply:
Eventually, yes.   We'll do a `grep` for Python 3.7 when we drop support for it, and should update this min version.